internal/util/strings.go

```diff
@@ -23,3 +25,8 @@ func Contains(list []string, s string) bool {
 	}
 	return false
 }
+
+// RandomString returns random string of length len
+func RandomString(len int) string {
```

Comment:
I think you could use https://golang.org/src/math/rand/rand.go?s=11556:11594#L341

Reply:
https://golang.org/pkg/math/rand/#Read

Reply:
I don't see how that is better than what is proposed.

Reply:
in general i would say it's better to use standard function from the go std package than rolling your own,

Reply:
I will get rid of this function. But still using the same body. `strconv.Itoa(rand.Intn(1000))`(go std package) is still better than `rand.Read(p []byte)` (why? https://syslog.ravelin.com/byte-vs-string-in-go-d645b67ca7ff)

@chmouel May be I am guilty of premature optimization here :stuck_out_tongue_winking_eye: